tests/fuzzer/fuzzer.py

```diff
@@ -388,7 +388,7 @@ def make_payload_path(infra: int, path: List[Tuple[str, float]]) -> Dict:
     :param path: List of steps
     :return: Dict
     """
-    path_payload = {"infra": infra, "name": "foo", "steps": [convert_stop(stop) for stop in path]}
+    path_payload = {"rolling_stocks": [102], "infra": infra, "name": "foo", "steps": [convert_stop(stop) for stop in path]}
```

Comment:
I'd prefer if we either use an empty list, or we query the available rolling stocks to pick a random one. This arbitrary ID sounds unstable.

This applies to the other test that uses this ID